Makefile

```diff
@@ -59,12 +59,19 @@ test/e2e: cluster/prepare
 	# operator-sdk test  local --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count" --namespace ${NAMESPACE} --up-local --debug --verbose ./test/e2e
 	go test -tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -mod=vendor ./test/e2e/... -root=$(PWD) -kubeconfig=$(HOME)/.kube/config -globalMan deploy/empty-init.yaml -namespacedMan deploy/empty-init.yaml -test.v -singleNamespace -parallel=1 -localOperator -test.timeout 0
 
-.PHONY: test/e2e-image
-test/e2e-image:
+.PHONY: test/e2e-latest-image
```

Comment:
@vmuzikar Could you please check if this doesn't break anything?

Reply:
Yes, there are no more references to `e2e-image`.